index.js

```diff
@@ -35,7 +36,6 @@ module.exports = (input, opts) => {
 		throw new Error('Could not use Yarn without yarn.lock file');
 	}
 
-	// TODO: remove sometime far in the future
```

Comment:
Don't do unrelated changes.

Reply:
:+1:  
Noticed after the commit but didn't want to make a commit just for it.